recipes/zarrtraj/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
+
```

Comment:
@ljwoods2 you don't need to vendor the license file, rather just provide the name of the license file in the repo.